README.md

```diff
@@ -5,56 +5,36 @@
 
 The open-source RChain project is building a decentralized, economic, censorship-resistant, public compute infrastructure and blockchain. It will host and execute programs popularly referred to as “smart contracts”. It will be trustworthy, scalable, concurrent, with proof-of-stake consensus and content delivery.
 
-[RChain Developer](https://developer.rchain.coop/) has a [Rholang tutorial](https://developer.rchain.coop/tutorial)
-and other information for building on top of the platform, including a roadmap of planned features and releases.
-This repository is the evolving platform itself.
+[RChain Developer](https://developer.rchain.coop/) features project-related tutorials and documentation, project planning information, events calendar, and information for how to engage with this project.
 
-### Communication
+## Building and running
 
-The `comm` subproject contains code for network related operations for RChain.
+Pre-release versions of the RChain software are now available. We plan to launch the full platform in Q4 of 2018.
 
-The network layer is the lowest level component in the architecture and it
-is featured in our **Node.Hello (v0.1) release**. The simplest way to get
-started is with [docker][]: `docker run -ti rchain/rchain-comm`. For other options,
-see [comm/README.md][cr].
+### Running from Docker
+Visit the [RChain Docker storefront](https://store.docker.com/profiles/rchain) for the most recent pre-release versions of the RChain software.
 
-[docker]: https://store.docker.com/community/images/rchain/rchain-comm
-[cr]: https://github.com/rchain/rchain/tree/master/comm
+### Building and running from source
+#### Prerequisites
+* Java Development Kit (JDK), version 8
+    - We recommend using the OpenJDK 
+    - Alternatively, the [Oracle JDK](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html) is an option
```

Comment:
I don't believe we have tested the system with Oracle JDK.  We need to validate that someone either in the community or on the Dev team has built and run the node successfully on the Oracle JDK